src/model.rs

```diff
@@ -1153,6 +1153,31 @@ impl Collections {
         }
     }
 
+    /// If a line name is empty, it's set with the name of its first "forward" route (in alphabetical order)
+    pub fn enhance_line_names(&mut self, lines_to_routes: &impl Relation<From = Line, To = Route>) {
+        let mut line_names: HashMap<Idx<Line>, String> = HashMap::new();
```

Comment:
very minor and quite picky but if I understand this correctly, this could be a `Vec<(Idx<Line>, String)>` no?

Reply:
Lighter indeed ! Done 06f0279a0bccb921596f676c8c1bbc376f74f0b3